### DISCUSSION TITLE
Ability to open all links in a list in multiple tabs

### DISCUSSION THREAD

**Discussion by milindpatel63** [Kovah/LinkAce#389] [Ideas & Feature Requests]
I use linkace as a collection for my bookmarks.

For example, I have made a collection of tech blogs that I browse daily.
I would love a feature to open all the links in a specific list in separate tabs in browser with a single click.
Maybe a icon next to list names in list view.

**Comment by milindpatel63** [comment]
> You can achieve this under All Lists. Right click on the list of interest and select open in new tab. Simple as that.

That opens the list description page with all the link in that list. But still no way to bulk open all those links.

**Comment by James000r** [comment]
That feature would indeed be very important to me. Thanks in advance for consideration Kovah. PS: We already have 7 Upvotes.

**Reply by Kovah** [reply]
Thanks for the hint, will have a look for this for LinkAce 2 which is already in Beta.

**Comment by Kovah** [comment]
I have tested this. Unfortunately, only Safari allowed to open more than one link in multiple tabs. When using Chrome or Firefox, the browsers probably detect this behaviour as spammy methods and block opening more than 1 link. 
The feature will be part of LinkAce v2, but probably needs additional browser configuration for most users.
